tests/integration/goldens/asset/samples/generated_samples/cloudasset_generated_asset_v1_asset_service_delete_feed_sync.py

```diff
@@ -35,6 +35,7 @@ def sample_delete_feed():
 
     # Initialize request argument(s)
     request = asset_v1.DeleteFeedRequest(
+        name="projects/{project}/feeds/{feed}",
```

Comment:
Same comment about `{project}` and `{feed}`

Reply:
Tracking a fix in https://github.com/googleapis/gapic-generator-python/issues/965